test-app/tests/unit/-private/properties/create-test.ts

```diff
@@ -94,7 +94,7 @@ module('create', function () {
       // @ts-expect-error violate types to check if it fails in weakly-typed envs
       create('');
       assert.true(false, 'should error');
-    } catch (e) {
+    } catch (e: any) {
```

Comment:
can we have `Error` instead of `any`?

Reply:
We can't use the type `Error` here, due to:

`Catch clause variable type annotation must be 'any' or 'unknown' if specified.ts(1196)`

This is an annoying issue with TypeScript, see a [long discussion thread here](https://github.com/microsoft/TypeScript/issues/20024). Alternatively, and probably more correct, we can type this as `unknown`.